civis/ml/_helper.py

```diff
@@ -0,0 +1,40 @@
+from civis import APIClient
+from civis.ml._model import _PRED_TEMPLATES
+
+
```

Comment:
If the `__init__` is doing an `import *`, then we should have this module control what gets imported.
```suggestion
__all__ = ['list_models']
```